Dockerfile

```
@@ -0,0 +1,41 @@
+FROM elixir:alpine
+#SHELL ["/bin/bash", "-c"]
+
+WORKDIR /root
+RUN apk --update add git nodejs nodejs-npm alpine-sdk
+
+RUN git clone https://github.com/wassuphq/wassup.git
+WORKDIR /root/wassup
+RUN mix local.hex --force
+RUN mix local.rebar --force
+RUN mix deps.get
```

```suggestion
RUN mix deps.get --only prod
```

as per https://github.com/wassuphq/wassup#production-setup.